spec/models/form526_submission_spec.rb

```diff
@@ -1288,6 +1288,20 @@ def expect_no_max_cfi_logged(diagnostic_code)
           end.to change(Form526ConfirmationEmailJob.jobs, :size).by(1)
         end
       end
+
+      context 'with disability_526_call_received_email_from_polling enabled' do
+        subject { create(:form526_submission, :with_multiple_succesful_jobs) }
+
+        before do
+          Flipper.enable(:disability_526_call_received_email_from_polling)
```

Comment:
feature flags in the test environment are automatically enabled.
did we mean to disable at the start, enable here, then disable in an `after` clause for the next tests to isolate these unit tests?
that's the usual pattern we follow.
and that may be why the tests are failing. i haven't looked at the details, but that seems like a solid guess 🤣

Reply:
Ah, actually I think you're right! Will try it out and make the adjustment